include/spore/codegen/codegen_cache.hpp

```diff
@@ -47,12 +50,21 @@ namespace spore::codegen
 
         bool check_and_update(const std::string_view& file)
         {
+            std::string file_content;
```

Comment:
If we don't need this change, can we revert it? I'd prefer the hash file interface then hash file content interface.

Reply:
done